v2/internal/runner/enumerate.go

```diff
@@ -22,8 +23,39 @@ func (r *Runner) runStandardEnumeration(executerOpts protocols.ExecuterOptions,
 	return r.executeTemplatesInput(store, engine)
 }
 
+// Get all the scan lists for a user/apikey.
+func (r *Runner) getScanList() error {
```

Comment:
Instead of keeping these here probably we should move these to nucleicloud subdirectory

Reply:
They are actually calling the actual functions in nuclei cloud. Not sure, if we want to move runnner methods to cloud?